src/components/search/search.tsx

```diff
@@ -141,31 +261,37 @@ const Search: React.FunctionComponent<SearchProps> = ({
                 </div>
             )}
             <div sx={{}}></div>
-            {!!numberOfResults || isValidating || error ? (
+            {!!getResultData().length || getResultIsValidating() || error ? (
```

Comment:
should we move this before the return and set it in a variable, that way we are not calling the function multiple times (also on line 267, line 280)

Reply:
Good call, I think I had it that way and can't recall why I changed this to a function. Will update.